class_vimrc.txt

```diff
@@ -0,0 +1,36 @@
+"~/.vimrc
+"THIS IS THE CLASS VIMRC
+set nocompatible              " be iMproved, required
+filetype plugin indent on     " required
+set laststatus=2              " Always show the statusline
+syntax on                     " syntax highlighting
+
+" highly recommend getting a better colorscheme too!
+set background=dark
+colorscheme default
+
+set expandtab               "Insert spaces instead of tabs in insert mode. Use spaces for indents
+set tabstop=4               "Number of spaces that a <Tab> in the file counts for
+set shiftwidth=4            "Number of spaces to use for each step of (auto)indent
+set autoindent              "Always set auto-indenting on"
+
+set hlsearch                "highlight search matches
+set incsearch               "highlight while typing
+
+set hidden                  "allow modified buffers to hide
+
+set number                  "Display line numbers incrementing
+"set number relativenumber  "Display line numbers in relative positions
```

Comment:
You should give relativenumber a try.  It makes it much easier to go like 4 or 38 lines up/down since the column shows how many lines away you are.  Much faster than say 755g.

It also makes it much easier to move/copy text.  If I want lines 3-5 above me, you `:-3,-5t.` to make a copy (m to move).  Again, faster than typing the line numbers or counting rows.  Personal taste though!

Reply:
I need to learn this! ok I've made the default to be relative line numbering in the class vimrc